src/electionguard/ballot.py

```diff
@@ -628,9 +628,10 @@ def hashed_ballot_nonce(self) -> Optional[ElementModQ]:
         """
 
         if self.nonce is None:
-            log_warning(
-                f"missing nonce for ballot {self.object_id} could not derive from null nonce"
-            )
+            if not isinstance(self, CiphertextAcceptedBallot):
```

Comment:
why dont we just get rid of the error message entirely? or change it to `log_debug`?  i think that's preferable over checking the subclassed type, right?  Open/Close Principle?

Reply:
I'm fine with nuking this entirely. If you're going to keep any sort of logging here, it should only happen with dealing with CiphertextBallots in other states. A CiphertextAcceptedBallot will never have a nonce in it.